frontend-react/src/components/header/ReportStreamHeader.tsx

```diff
@@ -50,8 +53,8 @@ export const ReportStreamHeader = () => {
 
         /* SENDERS ONLY */
         if (
-            permissionCheck(PERMISSIONS.SENDER, authState.accessToken) ||
-            permissionCheck(PERMISSIONS.PRIME_ADMIN, authState.accessToken)
+            memberships.state.active?.memberType === MemberType.SENDER ||
+            memberships.state.active?.memberType === MemberType.PRIME_ADMIN
```

Comment:
I need to go through and find all the rest of the old permissions implementations and swap them over. I _think_ I can refactor out another Utils file with this work.

Reply:
#5691 now open!